src/collar/oc_rlvextension.lsl

```diff
@@ -286,30 +282,32 @@ MenuSetValue(key kID, integer iAuth, string sValueName) {
 integer g_iLastOwnerEx;
 integer g_iLastTrustedEx;
 
-updateList(list newlist, integer type) // type 1=owner, 2=trusted;
+integer LIST_TYPE_OWNER = 1;
+integer LIST_TYPE_TRUSTED = 2;
+UpdateList(list newlist, integer type) ;
```

Comment:
All the rest of the functions in this file (and all of OC) use `UpperCamelCase()`.  Please don't introduce a new style (like `lowerCamelCase` unless we've agreed to it as a team and written it down in a style guide that we can send new devs to.